Cargo.toml

```diff
@@ -7,35 +7,37 @@ edition    = "2021"
 
 [workspace.dependencies]
 # common libs
-anyhow = "1.0"
+anyhow = "^1.0"
 bech32 = "^0.9"
-cosmwasm-std = "1.2"
-cosmwasm-schema = "1.2"
-cosmwasm-storage = "1.2"
-cw-ownable      = "0.5.0"
-cw-paginate-storage = { version = "2.2.0", git = "https://github.com/DA0-DA0/dao-contracts.git" }
+cosmwasm-std = "^1.5"
```

Comment:
Should use explicit versions for contracts so versions aren't ambiguous.

Reply:
Actually explicit versions are defined in Cargo.lock. By using "^" makes it easier for updating to latest patch version (0.0.x). `cw-nfts` does the same.